src/index.html

```diff
@@ -25,16 +25,20 @@
     <link rel="apple-touch-icon" sizes="180x180" href="apple-icon-180x180.png">
     <link rel="icon" type="image/png" sizes="192x192" href="android-icon-192x192.png">
     <link rel="manifest" href="/manifest.json">
-    <!-- Global site tag (gtag.js) - Google Analytics -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-157133795-1"></script>
+    <!-- Google Tag Manager -->
     <script>
-        window.dataLayer = window.dataLayer || [];
-        function gtag(){dataLayer.push(arguments);}
-        gtag('js', new Date());
-
-        gtag('config', 'UA-157133795-1');
+      (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```

Comment:
The subscriptions to app events rely on `gtag` in the `index.js`

If this is all working as expected, then I guess is OK.

Reply:
So for the project I am setting up new events to track based on what I need to measure and I think these are new to the other events we set up and for seAp I don't think we need the old ones. I should show you my master analytics plan at some point :)